### PR TITLE
[Relations]: add `strict: false` to API post

### PR DESCRIPTION
**Pull request by joshuaellis**
### What does it do?

* Adds docs for `useCallbackRef` a new hook in the CM
* Adds `strict:false` to relation API posting

### Why is it needed?

* Docs make people happy
* the false flag avoids reversing the array

### How to test it?

* automated testing

### Related issue(s)/PR(s)

* resolves CONTENT-864

### CODE REVIEW

File: packages/core/admin/admin/src/content-manager/components/EditViewDataManagerProvider/utils/cleanData.js [hunk 1]
```diff
@@ -135,13 +135,11 @@ const cleanData = ({ browserState, serverState }, currentSchema, componentsSchem
           }, []);
 
           cleanedData = {
+            options: {
```
Comment (ronronscelestes) [inline packages/core/admin/admin/src/content-manager/components/EditViewDataManagerProvider/utils/cleanData.js:138]: just wondering, those changes don't seem correlated to the use callback ref doc 👀

Comment (joshuaellis) [reply packages/core/admin/admin/src/content-manager/components/EditViewDataManagerProvider/utils/cleanData.js:138]: I changed the title, it's a bit of a joint PR – I know they should be separate PRs 😓

**Review by jhoward1994 — APPROVED** [review]
LGTM 👍🏻

**Review by gu-stav — APPROVED** [review]

**Review by ronronscelestes — COMMENTED** [review]
not sure it's caused by this PR but found this issue: 

https://user-images.githubusercontent.com/71838159/212071515-47098522-8919-436a-8ad1-9fb269915275.mov

PUT reorder:
`2 - 3 - 1`
after reloading:
`3 - 2 - 1`

PUT reorder: 
image

GET after reload: 
image

**Comment by joshuaellis** [comment]
^^^^^ we're tackling this issue outside of this PR

**Pull request merged**